eng/common/internal-feed-operations.ps1

```diff
@@ -0,0 +1,135 @@
+param(
```

Comment:
What are all of these new .ps1, .sh, etc. files?  These get copied over from arcade?

Reply:
Correct

Reply:
https://github.com/dotnet/arcade/commit/fd55b348227d4ed0dd0e09c3d236bcc69bbf0119
https://github.com/dotnet/arcade/commit/53ee76a101ee579118c88c5637c5197b76e41d31#diff-c733d1aa559947784028270444f63ed6